test-apps/display-test-app/src/backend/Backend.ts

```diff
@@ -11,15 +11,18 @@ import { BackendIModelsAccess } from "@itwin/imodels-access-backend";
 import { IModelsClient } from "@itwin/imodels-client-authoring";
 import { IModelHost, IModelHostOptions, LocalhostIpcHost } from "@itwin/core-backend";
 import {
+  AuthorizationClient,
   IModelReadRpcInterface, IModelTileRpcInterface, RpcInterfaceDefinition, RpcManager,
   SnapshotIModelRpcInterface,
 } from "@itwin/core-common";
 import { MobileHost, MobileHostOpts } from "@itwin/core-mobile/lib/cjs/MobileBackend";
+import { TestBrowserAuthorizationClient } from "@itwin/oidc-signin-tool";
 import { DtaConfiguration, getConfig } from "../common/DtaConfiguration";
 import { DtaRpcInterface } from "../common/DtaRpcInterface";
 import { FakeTileCacheService } from "./FakeTileCacheService";
 import { EditCommandAdmin } from "@itwin/editor-backend";
 import * as editorBuiltInCommands from "@itwin/editor-backend";
+// import { ServiceAuthorizationClient } from "@itwin/service-authorization";
```

Comment:
please remove commented out code.

Reply:
I left it there intentionally. The intent is to get this to work. However, right now it throws exceptions at run-time when this code is uncommented.